### PR TITLE
gh-120433: Mention ``chocolatey`` for installing llvm on Windows as an alternative option

### PR DESCRIPTION
**Pull request by Eclips4**
* Issue: gh-120433

### CODE REVIEW

**Comment by Eclips4** [comment]
cc @savannahostrowski
I guess there's something gone wrong with our CI. [Previously we decided](https://github.com/python/cpython/pull/118983) to ignore changes in *.md files to avoid wasting CI time, but actually, it's running!

**Review by vstinner — APPROVED** [review]
LGTM

**Comment by hugovk** [comment]
> I guess there's something gone wrong with our CI. [Previously we decided](https://github.com/python/cpython/pull/119147) to ignore changes in *.md files to avoid wasting CI time, but actually, it's running!

This is the run: https://github.com/python/cpython/actions/runs/9489036340

I don't know why, but it's because the workflow file used for the run doesn't contain the `'!**/*.md'` exceptions: https://github.com/python/cpython/actions/runs/9489036340/workflow

Which were added in https://github.com/python/cpython/pull/119147 and is still in `main`: https://github.com/python/cpython/blob/main/.github/workflows/jit.yml

**Comment by Eclips4** [comment]
> > I guess there's something gone wrong with our CI. [Previously we decided](https://github.com/python/cpython/pull/119147) to ignore changes in *.md files to avoid wasting CI time, but actually, it's running!
> 
> This is the run: https://github.com/python/cpython/actions/runs/9489036340
> 
> I don't know why, but it's because the workflow file used for the run doesn't contain the `'!**/*.md'` exceptions: https://github.com/python/cpython/actions/runs/9489036340/workflow
> 
> Which were added in #119147 and are still in `main`: https://github.com/python/cpython/blob/main/.github/workflows/jit.yml

I've created #120435 to try to fix that.
Actually, what do you mean when you say "but it's because the workflow file used for the run doesn't contain the '!**/*.md' exceptions:"? I'm seeing that:
```yml
name: JIT
on:
  pull_request:
    paths:
      - '!**/*.md'
      - '!**/*.ini'
      - '**jit**'
      - 'Python/bytecodes.c'
      - 'Python/optimizer*.c'
      - '!Python/perf_jit_trampoline.c'
  push:
    paths:
      - '!**/*.md'
      - '!**/*.ini'
      - '**jit**'
      - 'Python/bytecodes.c'
      - 'Python/optimizer*.c'
      - '!Python/perf_jit_trampoline.c'
```

**Comment by hugovk** [comment]
So it does... I missed them because the PR added them to the end of the lists, not the top: 🤔 

```diff
       - '**jit**'
       - 'Python/bytecodes.c'
       - 'Python/optimizer*.c'
+      - '!Python/perf_jit_trampoline.c'
+      - '!**/*.md'
+      - '!**/*.ini'
   push:
     paths:
       - '**jit**'
       - 'Python/bytecodes.c'
       - 'Python/optimizer*.c'
+      - '!Python/perf_jit_trampoline.c'
+      - '!**/*.md'
+      - '!**/*.ini'
```
I wonder why they're different.

**Review by savannahostrowski — APPROVED** [review]

File: Tools/jit/README.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ Homebrew won't add any of the tools to your `$PATH`. That's okay; the build scri
 
 Install LLVM 18 [by searching for it on LLVM's GitHub releases page](https://github.com/llvm/llvm-project/releases?q=18), clicking on "Assets", downloading the appropriate Windows installer for your platform (likely the file ending with `-win64.exe`), and running it. **When installing, be sure to select the option labeled "Add LLVM to the system PATH".**
 
+Alternatively, you can use [chocolatey](https://chocolatey.org):
+
+```sh
+choco install llvm --version=18.1.0
```
Comment (sobolevn) [inline Tools/jit/README.md]: Is it possible to specify just `--version=18`, without minor and patch releases? Other tools do not specify exact version.

Comment (Eclips4) [reply Tools/jit/README.md]: > Is it possible to specify just `--version=18`, without minor and patch releases? Other tools do not specify exact version.

Unfortunately, no.
We can also omit ``--version`` and the latest stable release will be installed. What do you think about that?

Comment (vstinner) [reply Tools/jit/README.md]: I prefer to stick to a version.

Comment (Eclips4) [reply Tools/jit/README.md]: So, here's the list of available versions:
```
llvm 18.1.6 [Approved] Downloads cached for licensed users
llvm 18.1.5 [Approved] Downloads cached for licensed users
llvm 18.1.4 [Approved] Downloads cached for licensed users
llvm 18.1.3 [Approved]
llvm 18.1.2 [Approved]
llvm 18.1.0 [Approved] Downloads cached for licensed users
```
Which one do you prefer? The latest one?

Comment (vstinner) [reply Tools/jit/README.md]: Latest, yes.

Comment (Eclips4) [reply Tools/jit/README.md]: Done

**Review by vstinner — APPROVED** [review]
LGTM.

**Review by sobolevn — APPROVED** [review]
We would have to update it from time to time, but there's no other way, I guess ;)

Thanks!

**Pull request merged**